spec/lib/modules/ai_bot/commands/setting_context_command.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+RSpec.describe DiscourseAi::AiBot::Commands::SettingContextCommand do
+  let(:bot_user) { User.find(DiscourseAi::AiBot::EntryPoint::GPT3_5_TURBO_ID) }
+  let(:command) { described_class.new(bot_user: bot_user, args: nil) }
+
+  # tests require that rg is installed and are likely going to fail in CI
```

Comment:
If you want to skip only on CI, you can do something like this in each test instead of xit:

```ruby
skip("rg is needed for these tests") if ENV["CI"]
```

Reply:
hmmm that is a good idea